src/mumble/mumble_pch.hpp

```diff
@@ -81,6 +81,9 @@
 #define interface struct
 #endif
 
+#ifndef _WIN32_WINNT
```

Comment:
In all internal builds and CI builds we use warnings-as-errors.
We strive to keep the build warning free.

User packages are not shipped with that option on, because we can't predict the future.
But we try our best to ensure that we don't ship anything to end-users that will trigger warnings.